pkg/scheduler/framework/plugins/noderesources/fit_test.go

```diff
@@ -587,18 +587,11 @@ func TestStorageRequests(t *testing.T) {
 		name       string
 		wantStatus *framework.Status
 	}{
-		{
-			pod: newResourcePod(framework.Resource{MilliCPU: 1, Memory: 1}),
-			nodeInfo: framework.NewNodeInfo(
-				newResourcePod(framework.Resource{MilliCPU: 10, Memory: 10})),
-			name:       "due to container scratch disk",
-			wantStatus: framework.NewStatus(framework.Unschedulable, getErrReason(v1.ResourceCPU)),
-		},
 		{
 			pod: newResourcePod(framework.Resource{MilliCPU: 1, Memory: 1}),
 			nodeInfo: framework.NewNodeInfo(
 				newResourcePod(framework.Resource{MilliCPU: 2, Memory: 10})),
-			name: "pod fit",
+			name: "empty storage requested, and pod fits",
```

Comment:
```suggestion
			name: "pod fits",
```

We required mem here.

Reply:
ditto.
The function name here is TestStorageRequests, so I changed the line with more info.

Reply:
sg